### ISSUE TITLE
RetinaFace tensorflow 2.0+ implementation

### ISSUE THREAD

**Issue by StanislasBertrand** [deepinsight/insightface#1184]
Hello,

First of all thank you for your great work on face algorithms that has been very useful to me over the years.

As I wanted to use retinaface in tf and couldnt find a satisfactory implementation, I recently converted your model to tensorflow2.0+ :
https://github.com/StanislasBertrand/RetinaFace-tf2

I have achieved almost the same accuracy on widerface, though inference is a little bit slower.

Since there are quite a few questions about tf/keras implementations being available, I thought I could leave this issue here. If this is against your policy, please let me know and I will remove the issue.
